hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTimeline.java

```diff
@@ -187,6 +187,12 @@
    */
   Stream<HoodieInstant> getInstants();
 
+  /**
+   * @return Get the stream of completed instants in reverse order
+   * TODO Change code references to getInstants() that reverse the instants later on to use this method instead.
```

Comment:
@bhasudha can you file a ticket for this work for later ? and may be assign a "newbie" component